packages/spor-react/src/theme/components/alert-service.ts

```diff
@@ -46,7 +37,50 @@ const config = helpers.defineMultiStyleConfig({
       paddingX: "0.75rem",
       paddingTop: "0.375rem",
       paddingBottom: "0.9375rem",
-      color: "white",
+    },
+  },
+  variants: {
+    "global-deviation": {
+      container: {
+        _hover: {
+          outlineColor: "primrose",
+        },
+        _active: {
+          backgroundColor: "blonde",
+          outlineColor: "primrose",
+        },
+      },
+      outerBox: {
+        outlineColor: "primrose",
+        backgroundColor: "blonde",
+      },
+      notificationText: {
+        color: "black",
+      },
+      serviceMessageContent: {
+        color: "black",
+      },
+    },
+    service: {
+      container: {
+        _hover: {
+          outlineColor: "blueGreen",
+        },
+        _active: {
+          backgroundColor: "pine",
+          outlineColor: "pine",
+        },
+      },
+      outerBox: {
+        outlineColor: "blueGreen",
+        backgroundColor: "darkTeal",
+      },
+      notificationText: {
+        color: "white",
+      },
+      serviceMessageContent: {
```

Comment:
Hvis du kan legge til flere styling-objekter her, så mangler du vel stiler for title og icon (ref. kommentar over)